### DISCUSSION TITLE
changing pool plots in remote harvesters?

### DISCUSSION THREAD

**Discussion by nicezic** [Chia-Network/chia-blockchain#7241] [Q&A]
If I change pool settings in the main farming( within a private key ) machine, are the NFT plots in the remote harvesters applied?

**Comment by gryan315** [comment]
The pool data is stored on the NFT itself in the blockchain. A remote harvester doesn't even see the blockchain itself, it just sends plot info to the full node on the farmer which should handle the blockchain.
